meinberlin/apps/plans/forms.py

```diff
@@ -36,3 +38,31 @@ class Meta:
                 'required': _('Please locate the plan on the map.')
             }
         }
+
+
+class CustomMultipleChoiceField(forms.ModelMultipleChoiceField):
+
+    def clean(self, value):
+        value = [value]
+        return super().clean(value)
+
+
+class ProjectPlansDashboardForm(ProjectDashboardForm):
+    plans = CustomMultipleChoiceField(
+        widget=forms.RadioSelect,
```

Comment:
Maybe we can have setting the widget inside the CustomMultipleChoiceField

Reply:
Yes, just change it!